src/Components/Render.cpp

```diff
@@ -0,0 +1,30 @@
+#include "Render.h"
```

Comment:
Please order the include according to the style guide:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes